packages/core/src/connectors/injected.ts

```diff
@@ -39,7 +41,7 @@ export class InjectedConnector extends Connector<
 > {
   readonly id: string
   readonly name: string
-  readonly ready = typeof window != 'undefined' && !!window.ethereum
+  readonly ready: boolean = false
```

Comment:
Probably don't need a default now since we assign in the constructor.

```suggestion
  readonly ready: boolean
```